<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Table</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="documentBttn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Document</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classificationBttn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Classification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logBttn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accountBttn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
